src/core_plugins/kibana/public/visualize/editor/sidebar.html

```diff
@@ -14,10 +14,14 @@
         <!-- tabs -->
         <ul class="nav navbar-nav">
           <li ng-class="{active: sidebar.section == 'data'}" ng-show="sidebar.showData">
-            <a class="navbar-link active" ng-click="sidebar.section='data'">Data</a>
+            <a class="vis-editor-subnav-link"
+               ng-class="{'is-vis-editor-sub-nav-link-selected': sidebar.section == 'data'}"
+               ng-click="sidebar.section='data'">Data</a>
```

Comment:
I'm checking some of the other .html files and i cant find this anywhere (last > in a seperate line) ...
also my editor (webstorm) wont put > in line with <a (no indent) if i put it in a seperate line .... 
would it be possible to configure .eseditorconfig and .eslint to follow this style ? 

also could you update [html style guide](https://github.com/elastic/kibana/blob/14f3bf0ed828ae043d8466ab4de05535f168737d/style_guides/html_style_guide.md) with this information: